python/graphstorm/gconstruct/remap_result.py

```diff
@@ -77,7 +158,7 @@ def worker_remap_node_pred(pred_file_path, nid_path, ntype,
         output_fname = f"{output_fname_prefix}_{pad_file_index(i)}.parquet"
 
         start = i * chunk_size
-        end = (i + 1) * chunk_size if i + 1 < chunk_size else len(pred_result)
+        end = (i + 1) * chunk_size if i + 1 < num_chunks else len(pred_result)
```

Comment:
this is a bug?

Reply:
why the previous tests cannot find the bug?

Reply:
This is resolved. It is a bug, but as the default value of chunk_size is very large, the bug was not triggered.